### ISSUE TITLE
Proposed API is only available when running out of dev or with the following command line switch

### ISSUE THREAD

**Issue by shiraze** [kdcro101/vscode-favorites#51]
**Describe the bug**
I now see the following warning when launching VSCode:
[kdcro101.favorites]: Proposed API is only available when running out of dev or with the following command line switch: --enable-proposed-api kdcro101.favorites

**To Reproduce**
Launch VSCode when a list of favorites has been set up

**Expected behavior**
I would expect to see list of favorites in left pane

**Screenshots**
![image](https://user-images.githubusercontent.com/9248719/87432173-8f40f800-c5df-11ea-8a8b-a5f4f0fb0fb3.png)
![image](https://user-images.githubusercontent.com/9248719/87432254-a8e23f80-c5df-11ea-9521-81475a448c01.png)

**System info (please complete the following information):**
 - OS version: Windows 10
 - Favorites extension version: v2.4.5
 - vscode version
Version: 1.47.0 (user setup)
Commit: d5e9aa0227e057a60c82568bf31c04730dc15dcd
Date: 2020-07-09T08:02:06.629Z
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18362

**Additional info**
I'm not sure where the list of favorites files are stored.
I only see this issue on VSCode projects that make use of favorites.

**Comment by shiraze** [comment]
Found the "favorites" file - stored as .favorites.json in root folder for project.
This is contents of file:
```
[
    {
        "type": "File",
        "name": "c:\\IS Transfer\\Source\\React\\react\\public\\appSettings.js",
        "parent_id": null,
        "fsPath": "c:\\IS Transfer\\Source\\React\\react\\public\\appSettings.js",
        "workspaceRoot": null,
        "workspacePath": null,
        "id": "J741zfpQAMCU5grI"
    },
    {
        "type": "File",
        "name": "c:\\IS Transfer\\Source\\React\\react\\package.json",
        "parent_id": null,
        "fsPath": "c:\\IS Transfer\\Source\\React\\react\\package.json",
        "workspaceRoot": null,
        "workspacePath": null,
        "id": "LA5hepTq19MmnEFv"
    },
    {
        "type": "File",
        "name": "c:\\Windows\\System32\\drivers\\etc\\hosts",
        "parent_id": null,
        "fsPath": "c:\\Windows\\System32\\drivers\\etc\\hosts",
        "workspaceRoot": null,
        "workspacePath": null,
        "id": "ITXtNfRK8hCOeqP6"
    }
]
```

**Comment by jeromeboivin** [comment]
I have the same issue since VSCode upgraded to version 1.47.0

**Comment by jeromeboivin** [comment]
You must have this line in the package.json file of your extension: "enableProposedApi": true.
[https://code.visualstudio.com/updates/v1_27#_proposed-extension-apis](https://code.visualstudio.com/updates/v1_27#_proposed-extension-apis)

**Comment by csicky** [comment]
Came here to report this.

**Comment by shiraze** [comment]
> You must have this line in the package.json file of your extension: "enableProposedApi": true.
> https://code.visualstudio.com/updates/v1_27#_proposed-extension-apis

I think that setting is only needed if the extension is using proposed api functions. I don't think this extension has changed in a long time, so I think VSCode is probably to blame.

**Comment by shiraze** [comment]
https://github.com/microsoft/vscode/issues/102530

**Comment by shiraze** [comment]
This is supposed to be fixed in VS Code 1.47.2

**Comment by shiraze** [comment]
I can confirm that VS Code 1.47.2 has fixed the issue. I now see Favorites and no warning:
![image](https://user-images.githubusercontent.com/9248719/87879141-be40da80-c9e0-11ea-9f15-f005fd90f8c4.png)
